.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,15 @@
+{
+    "configurations": [
```

Comment:
Do we need macOS and Windows configs too? Or will this config work fine on any OS?

Reply:
It will only work on Linux. I unfortunately can't test on Windows or macOS.